mobile/js_files/package.json

```diff
@@ -33,6 +34,7 @@
     "react-native-languages": "^3.0.2",
     "react-native-mail": "git+https://github.com/status-im/react-native-mail.git#v4.0.0-status",
     "react-native-navigation-twopane": "git+https://github.com/status-im/react-native-navigation-twopane.git#v0.0.2-status",
+    "react-native-os": "^1.2.5",
```

Comment:
@jakubgs why is `react-native-os` reappearing in the mobile `package.json`?

Reply:
Good question, probably because it was created before that was removed. Let me fix that.